SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+| `URelativeSphereConstraint` | Includes all Actors within a sphere centered on the Actor that has `ActorInterestComponent`. |
+| `URelativeCylinderConstraint` | Includes all Actors within a cylinder centered on the Actor that has `ActorInterestComponent`. |
+| `URelativeBoxConstraint` | Includes all Actors within a bounding box centered on the Actor that has `ActorInterestComponent`. |
+| `UCheckoutRadiusConstraint` | Includes all Actors of a class or a derived class within a cylinder centered on the Actor that has `ActorInterestComponent`. |
+| `UActorClassConstraint` | Includes all Actors of a class. You can optionally include derived classes. |
+| `UComponentClassConstraint` | Includes all Actors with an Unreal Actor component of a specified class. You can optionally include derived classes. |
+
+For more information, see the [interest constraints header file](https://github.com/spatialos/UnrealGDK/blob/0.6.0-rc/SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialInterestConstraints.h).
+
+To attach this component to an Actor, open the Actor's Blueprint and, on the Components tab, select **Add Component**. Actor Interest is in the SpatialGDK section:
+
+![Add component]({{assetRoot}}assets/screen-grabs/game-client-interest-management/add-component.png)
+
+### Example
+
+![Set up interest]({{assetRoot}}assets/screen-grabs/game-client-interest-management/set-up-interest.png)<br>
+_Image: The Interest section of the details panel for `ActorInterestComponent`_
+
+In the screenshot above, `ActorInterestComponent` has two queries, so the game client receives updates about:
+
+* all Pawns within 200 meters of the Actor that has `ActorInterestComponent` 
+* all Actors of the class `Landmark` or derived from the class `Landmark`
+
+## AlwaysInterested
+
+You might want a game client to always receive updates about some Actors, regardless of the positions of the player and of these Actors. For example, you might want the game client to always receive updates about an important landmark, or about other members of the player’s team. You can use the `AlwaysInterested` `UPROPERTY` specifier for this.
+
+An `AlwaysInterested` `UPROPERTY` must:
+
+* be an object reference (either an AActor or UObject)
+* have a [`Replicated`](https://docs.unrealengine.com/en-US/Gameplay/Networking/Actors/Properties/index.html) or [`Handover`]({{urlRoot}}/content/actor-handover) specifier
+
+You set up an `AlwaysInterested` `UPROPERTY` on a game client’s PlayerController, or on any other Actor that the game client might own, and then you specify the Actor that you want the game client to always receive updates about.
+
+### Example
+
+For example, you might want a game client to always receive updates about a team base:
+
+```
+UPROPERTY(Replicated, AlwaysInterested)
+AActor* TeamBase
+```
+
+If TeamBase is a valid Actor reference, then the game client receives updates about that Actor, regardless of TeamBase’s position in the game world.
```

Comment:
Ah there's a subtlety here regarding the Actor reference pointing to a replicated Actor, otherwise it won't work. Lets also talk about this.